bin/changelog/config.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const pkg = require( '../../package.json' );
+
+const REPO = pkg.repository.url
+	// remove https://github.com:
+	.split( ':' )[ 2 ]
+	// remove the .git ending.
+	.slice( 0, -4 );
+
+if ( pkg.changelog === undefined ) {
```

Comment:
Something that my script was missing is merging defaults since someone can define only one thing like `labelPrefix` and not define the rest, something like [`wp_parse_args`](https://developer.wordpress.org/reference/functions/wp_parse_args/)

Reply:
Something like this might be useful for this: https://www.npmjs.com/package/merge-config (with the bonus of working with command-line arguments and environment variables).